books/free-programming-books-fr.md

```diff
@@ -240,6 +241,11 @@
 * [Apprendre Meteor](https://mquandalle.gitbooks.io/apprendre-meteor/content/) - Maxime Quandalle
 
 
+### Pascal - Object Pascal
+
+* [J'apprends à programmer en Pascal Objet](https://lazarus.developpez.com/livre/livre-pascal-objet/) - Gilles Vasseur, Jean-Luc Gofflot (PDF, EPUB) (email address required)
```

Comment:
as I explained in a previous PR, we won't accept this link because it requires an working email. remove it.

Reply:
Done. I asked the authors to put it somewhere else and will resubmit it if they do. Thanks